x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/VerifierTests.java

```diff
@@ -2705,26 +2705,16 @@ private void checkFullTextFunctionsInStats(String functionInvocation) {
     }
 
     public void testVectorSimilarityFunctionsNullArgs() throws Exception {
-        if (EsqlCapabilities.Cap.COSINE_VECTOR_SIMILARITY_FUNCTION.isEnabled()) {
-            checkVectorFunctionsNullArgs("v_cosine(null, vector)");
-            checkVectorFunctionsNullArgs("v_cosine(vector, null)");
-        }
-        if (EsqlCapabilities.Cap.DOT_PRODUCT_VECTOR_SIMILARITY_FUNCTION.isEnabled()) {
-            checkVectorFunctionsNullArgs("v_dot_product(null, vector)");
-            checkVectorFunctionsNullArgs("v_dot_product(vector, null)");
-        }
-        if (EsqlCapabilities.Cap.L1_NORM_VECTOR_SIMILARITY_FUNCTION.isEnabled()) {
-            checkVectorFunctionsNullArgs("v_l1_norm(null, vector)");
-            checkVectorFunctionsNullArgs("v_l1_norm(vector, null)");
-        }
-        if (EsqlCapabilities.Cap.L2_NORM_VECTOR_SIMILARITY_FUNCTION.isEnabled()) {
-            checkVectorFunctionsNullArgs("v_l2_norm(null, vector)");
-            checkVectorFunctionsNullArgs("v_l2_norm(vector, null)");
-        }
+        checkVectorFunctionsNullArgs("v_cosine(null, vector)");
+        checkVectorFunctionsNullArgs("v_cosine(vector, null)");
+        checkVectorFunctionsNullArgs("v_dot_product(null, vector)");
+        checkVectorFunctionsNullArgs("v_dot_product(vector, null)");
+        checkVectorFunctionsNullArgs("v_l1_norm(null, vector)");
+        checkVectorFunctionsNullArgs("v_l1_norm(vector, null)");
+        checkVectorFunctionsNullArgs("v_l2_norm(null, vector)");
+        checkVectorFunctionsNullArgs("v_l2_norm(vector, null)");
+        checkVectorFunctionsNullArgs("v_magnitude(null)");
         if (EsqlCapabilities.Cap.MAGNITUDE_SCALAR_VECTOR_FUNCTION.isEnabled()) {
-            checkVectorFunctionsNullArgs("v_magnitude(null)");
-        }
-        if (EsqlCapabilities.Cap.HAMMING_VECTOR_SIMILARITY_FUNCTION.isEnabled()) {
```

Comment:
I think something is not right here - shouldn't v_magnitude tests still be skipped for non snapshot builds and the v_hamming ones be executed without a cap check?

Reply:
Oh I screwed up here, thanks for the catch!

I'm fixing in 21f8197517770ee1a5b28bee94d1776856254c50, and adding the `test-release` label